Formula/couchdb.rb

```diff
@@ -6,6 +6,11 @@ class Couchdb < Formula
   sha256 "8bea574faa6bb0926c670542d8318c322268cf7e6614dab318dea734ccf1b00c"
   license "Apache-2.0"
 
+  livecheck do
+    url :homepage
+    regex(/Couch.+?href=.*?v?(\d+(?:\.\d+)+)\.t/im)
```

Comment:
```suggestion
    regex(/href=.*?apache-couchdb[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

The tarball URL in the `homepage` source is the same as the `stable` URL (i.e., `https://www.apache.org/dyn/closer.lua?path=/couchdb/source/3.2.0/apache-couchdb-3.2.0.tar.gz`), so we can use the typical regex format for matching a file link in an HTML page.

Reply:
Oh, I see. Make sense.